src/Resources/config/services.yml

```diff
@@ -80,6 +80,56 @@ services:
         calls:
             - ["setContainer", ["@service_container"]]
 
+    contao.menu.matcher:
+        class: Knp\Menu\Matcher\Matcher
+        public: false
+
+    contao.menu.picker_menu_builder:
+        class: Contao\CoreBundle\Menu\PickerMenuBuilder
```

Comment:
The alias is built from the class name, so `Menu\PickerMenuBuilder` becomes `menu.picker_menu_builder`. That's the regular Symfony behavior I think.